webpack.config.js

```diff
@@ -42,17 +43,31 @@ function afterEmit(root, functions, provider) {
   return {
     apply: (compiler) => {
       compiler.hooks.afterEmit.tap('Functions Deployment', async () => {
-        Promise.all(
-          functions.map((item) =>
-            item.filename.endsWith('.zip') && provider.createOrUpdateFunction(path.parse(item.filename).name, item.content).catch((x) => console.error(x))
-          )
-        ).then(() => {
-          const redirects = {}
-          // compilation.entrypoints.forEach((_, key) => (redirect[filename] = createRedirect(filename)))
-          // fs.writeFileSync(resolve(root, 'public', 'functions-redirects.json', `${JSON.stringify(redirects)}\n`)
-        })
+        const urls = await Promise.all(
```

Comment:
You could turn the logic inside this promise all into a separate function to keep this part of the code clearer